test/integration/recommendPage.js

```diff
@@ -76,7 +76,7 @@ describe('Recommend page', () => {
     iExpect.htmlWith200Status(res);
     const $ = cheerio.load(res.text);
 
-    expect($($('.page-section p')[3]).text()).to.equal('This is because you’re under the legal age of consent for sex.');
+    expect($($('.page-section p')[3]).text()).to.equal('We\'re not able to offer test kits if you\'re under 16. You\'ll need to see a health professional to get tested.');
```

Comment:
Rather than use two $ wrappers, you could use:
`$('.page-section p').eq(3).text()`